### PR TITLE
Added new play called infinite scroll with debouncing

### PR DESCRIPTION
**Pull request by WhyNotSantosh**
> First thing, PLEASE READ THIS: [ReactPlay Code Review Checklist](https://github.com/reactplay/react-play/wiki/ReactPlay-Code-Review-Checklist)

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.

# Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

# Screenshots or example output

### CODE REVIEW

File: src/plays/tube2tunes/Tube2tunes.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ import Loader from './components/Loader';
 import TubeTunesLogo from './components/tubeTuneLogo.svg';
 
 // WARNING: Do not change the entry componenet name
-function Tube2tunes(props) {
+function Tube2Tunes(props) {
```
Comment (niteshseram) [inline src/plays/tube2tunes/Tube2tunes.js:10]: Hi @WhyNotSantosh , I think we need to either rename this file to `Tube2Tunes.js` or change the export in `src/plays/index.js` file from `export { default as Tube2Tunes } from 'plays/tube2tunes/Tube2Tunes';` to `export { default as Tube2Tunes } from 'plays/tube2tunes/Tube2tunes';`.
I have tried to fix this same issue in my below PR as well. But if you want to fix this issue in this PR instead, then I can remove it from my PR.
https://github.com/reactplay/react-play/pull/1088

File: src/plays/tube2tunes/Tube2Tunes.js [hunk 2]
```diff
@@ -154,4 +154,4 @@ function Tube2tunes(props) {
   );
 }
 
-export default Tube2tunes;
+export default Tube2Tunes;
```
Comment (atapas) [inline src/plays/tube2tunes/Tube2Tunes.js:157]: You don't need this anymore. Please revert.

**Review by atapas — CHANGES_REQUESTED** [review]

File: src/plays/tube2tunes/Tube2Tunes.js (hunk 2 above)
Comment (WhyNotSantosh) [reply src/plays/tube2tunes/Tube2Tunes.js]: @atapas - reverted renaming changes.

**Comment by WhyNotSantosh** [comment]
@atapas - removed the diff

**Review by atapas — COMMENTED** [review]
Thanks for taking care.

**Comment by atapas** [comment]
@WhyNotSantosh, there are a bunch of lint errors. Please follow these steps to fix them and push

> https://github.com/reactplay/react-play#-format-and-lint-the-code



image

**Comment by WhyNotSantosh** [comment]
@atapas - let me know if this needs more changes

**Comment by atapas** [comment]
@WhyNotSantosh 

There is still one more

image

**Comment by WhyNotSantosh** [comment]
> @WhyNotSantosh
> 
> There is still one more
> 
> image

**Comment by WhyNotSantosh** [comment]
@atapas - fixed other two isues

**Comment by WhyNotSantosh** [comment]
> > @WhyNotSantosh
> > There is still one more
> > image

**Comment by atapas** [comment]
@WhyNotSantosh All fixed?

**Comment by WhyNotSantosh** [comment]
> @WhyNotSantosh All fixed?

@atapas - Yes

**Review by atapas — CHANGES_REQUESTED** [review]
@WhyNotSantosh Great to see this one passed the build finally.

A few comments:

- The new search initiates only after we make the textbox blank, not on every character type. Can you please add a note so that users know about this expectations
- The app needs improvements on responsive mode. Let the cards be aligned properly for better readability
image

**Comment by atapas** [comment]
> @WhyNotSantosh Great to see this one passed the build finally.
> 
> A few comments:
> 
> * The new search initiates only after we make the textbox blank, not on every character type. Can you please add a note so that users know about this expectations
> * The app needs improvements on responsive mode. Let the cards be aligned properly for better readability
>   image

@WhyNotSantosh did you get a chance to check this one?

**Comment by WhyNotSantosh** [comment]
> * The new search initiates only after we make the textbox blank, not on every character type. Can you please add a note so that users know about this expectations - FIXED - Search initiates immediately on change of text
> * The app needs improvements on responsive mode. Let the cards be aligned properly for better readability - FIXED

- There is one issue with thumbnail. As the title is bigger, the thumbnail is not occupying entire space. Screenshot below. This  requires css change which impacts entire repo or my title to be short.

image

**Comment by WhyNotSantosh** [comment]
@atapas - Tapas, any changes required from my side for this play?

**Comment by WhyNotSantosh** [comment]
@atapas - I received mail that this PR will be at stale soon. Any changes required from my side?

**Comment by atapas** [comment]
> Inspect

The problem is the branch got linting errors.

Can you please run the lint commands mentioned in the readme and push? Will merge it after testing

image

**Comment by WhyNotSantosh** [comment]
> ease follow these steps to fix them an

@atapas - Fixed lint errors

**Comment by atapas** [comment]
> > ease follow these steps to fix them an
> 
> @atapas - Fixed lint errors

@WhyNotSantosh was trying reaching you on discord

**Comment by WhyNotSantosh** [comment]
> > > ease follow these steps to fix them an
> > 
> > 
> > @atapas - Fixed lint errors
> 
> @WhyNotSantosh was trying reaching you on discord

@atapas - replied on discord

**Review by atapas — APPROVED** [review]

**Pull request merged**